build.yaml

```diff
@@ -347,6 +349,7 @@ common_cli:
     # bootstrapping. These are copied from the Jenkins Master /etc/slave_config
     # directory.
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE=/tmp/acceptance.yaml
+    export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_SECTION=storage-drivers.gce
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER=gce
```

Comment:
Yeah, I was tempted to get rid of it and instead introduce  `FUNCTIONAL_TEST_MINIMUM_ALLOCATABLE_SIZE` and `FUNCTIONAL_TEST_ALLOCATION_UNIT` which again, would help 3rd party driver developers to use our factory functions.